site/class-3-clean/wesselversluis/KNMI_20171003_hourly.txt

```diff
@@ -0,0 +1,133 @@
+# BRON: KONINKLIJK NEDERLANDS METEOROLOGISCH INSTITUUT (KNMI)

```

Comment:
Can you remove this file?

You can remove files through the github interface, by going to your fork, your branch, and the finding your file. Finally, click on the trash icon in the top-right.